rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/IGraphConfigurationService.java

```diff
@@ -66,7 +65,7 @@
     @GET
     @Path("/operations")
     @ApiOperation(value = "Gets all operations supported by the store. See <a href='https://github.com/gchq/Gaffer/wiki/operation-examples' target='_blank' style='text-decoration: underline;'>Wiki</a>.", response = Class.class, responseContainer = "list")
-    Set<Class<? extends Operation>> getOperations();
+    Set<Class> getOperations();
```

Comment:
I don't think REST API methods should return generic types - you get warnings when the REST API starts up if any are methods that return a generic type.